gt/main.c

```diff
@@ -81,39 +260,83 @@ gt_proc(void *arg)
 			continue;
 
 		for (i = 0; i < num_rx; i++) {
+			int ret;
+			uint8_t priority;
```

Comment:
You will no longer need variable `priority` since it's now part of `struct gt_packet_headers`.